### PR TITLE
[2.6]Adding the project rbac test cases to the existing rbac suite

### PR DESCRIPTION
**Pull request by anupama2501**
Back port of https://github.com/rancher/rancher/pull/39949

### CODE REVIEW

File: tests/v2/validation/rbac/rbac_test.go [hunk 1]
```diff
@@ -162,28 +204,71 @@ func (rb *RBTestSuite) ValidateNS(role string) {
...
 
-	//Testcase8 Remove added cluster member from the cluster as an admin
+	//Testcase10a Remove added cluster member from the cluster as an admin
 	err := users.RemoveClusterRoleFromUser(rb.client, rb.standardUser)
 	require.NoError(rb.T(), err)
+}
+
+func (rb *RBTestSuite) ValidateRemoveProjectRoles() {
+
+	//Testcase10b Remove added cluster member from the cluster as an admin
```
Comment (slickwarren) [inline tests/v2/validation/rbac/rbac_test.go:269]: just a nit, I think this comment is wrong / copied from the previous function

File: tests/v2/validation/rbac/rbac_test.go [hunk 2]
```diff
@@ -162,28 +204,71 @@ func (rb *RBTestSuite) ValidateNS(role string) {
...
+	case roleProjectMember:
+		require.Error(rb.T(), err)
 	}
 }
 
-func (rb *RBTestSuite) ValidateRemoveClusterRoles(role string) {
+func (rb *RBTestSuite) ValidateRemoveClusterRoles() {
 
-	//Testcase8 Remove added cluster member from the cluster as an admin
+	//Testcase10a Remove added cluster member from the cluster as an admin
```
Comment (slickwarren) [inline tests/v2/validation/rbac/rbac_test.go:262]: do we need a log for 10a vs. 10b?

**Review by slickwarren — DISMISSED** [review]
LGTM just some comments

File: tests/v2/validation/rbac/rbac_test.go (hunk 1 above)
Comment (anupama2501) [reply tests/v2/validation/rbac/rbac_test.go]: Updated

File: tests/v2/validation/rbac/rbac_test.go (hunk 2 above)
Comment (anupama2501) [reply tests/v2/validation/rbac/rbac_test.go:262]: Line 437 will log for both project and cluster roles.

**Review by igomez06 — APPROVED** [review]
LGTM

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Pull request merged**